pptxProject.py

```diff
@@ -0,0 +1,141 @@
+import asyncio
+import os
+import time
+
+import aiohttp
+from flask import json
+from pptx import Presentation
+import openai
+
+
+REQ = "Please provide an English summary and brief explanation of the text on this slide, " \
```

Comment:
Don't use abbreviations in variable names -
https://softwareengineering.stackexchange.com/questions/72593/how-to-abbreviate-variable-names
Try to avoid i and j as well if there is a better name available.